src/legacy/core_plugins/kibana/public/visualize/wizard/type_selection/vis_help_text.tsx

```diff
@@ -17,40 +17,34 @@
  * under the License.
  */
 
-import { FormattedMessage } from '@kbn/i18n/react';
 import React from 'react';
 
 import { EuiSpacer, EuiText, EuiTitle } from '@elastic/eui';
 
-import { VisType } from 'ui/vis';
-
 interface VisHelpTextProps {
-  visType: VisType;
+  name: string;
+  title: string;
+  description?: string;
+  highlightMsg?: string;
 }
 
-export const VisHelpText = ({ visType }: VisHelpTextProps) => {
+export const VisHelpText = ({ name, title, description, highlightMsg }: VisHelpTextProps) => {
   return (
     <React.Fragment>
       <EuiTitle size="s">
-        <h2>{visType.title}</h2>
+        <h2>{title}</h2>
       </EuiTitle>
       <EuiSpacer size="s" />
-      <div id={`visTypeDescription-${visType.name}`}>
-        {visType.stage === 'experimental' && (
+      <div id={`visTypeDescription-${name}`}>
+        {highlightMsg && (
           <React.Fragment>
             <EuiText>
-              <em>
-                <FormattedMessage
-                  id="kbn.visualize.newVisWizard.experimentalDescription"
-                  defaultMessage="This visualization is experimental. The design and implementation
-                    are less mature than stable visualizations and might be subject to change."
-                />
-              </em>
+              <em>{highlightMsg}</em>
             </EuiText>
             <EuiSpacer size="s" />
           </React.Fragment>
         )}
-        <EuiText>{visType.description}</EuiText>
+        <EuiText>{description}</EuiText>
```

Comment:
I would refactor this so you don't need to declare two EuiText components like so:

```
<EuiText>
  {highlightMsg && (
    <p><em>{highlightMsg}</em></p>
  )}
  <p>{description}</p>
</EuiText>
```